src/Core/Grid/Query/SeoUrlsQueryBuilder.php

```diff
@@ -77,6 +80,22 @@ private function getQueryBuilder(array $filters)
                 'm.`id_meta` = l.`id_meta`'
             );
 
+        foreach ($filters as $name => $value) {
+            if ('id_lang' === $name) {
```

Comment:
@sarjon I saw the same usage in EmailController but the filters array by default does not have id_lang nor id_shop keys- should I define them by myself in getDefaults() function under SeoUrlsFilters class or maybe I just can't find anywhere where I can already take it and use it with the grid itself? :thinking:

Reply:
first, you cant define dynamic values (`id_lang` and `id_shop`) in `getDefaults()`. secondly, im not sure if i understand an issue. what is your use case?

Reply:
yeah I guess that's an option - I'm just wondering from where id_lang key comes from in _EmailLogsQueryBuilder_ line 80 ? :thinking:

Reply:
ohh, it comes from db, it's lang id in which email is sent.

Reply:
right I see - In my situation I need context language id and context shop id in order to reduce the results displayed - should I fetch these values in constructor of QueryBuilder or should I pass them in the filters array?

Reply:
inject, here's an example https://github.com/PrestaShop/PrestaShop/pull/10160/files#diff-d096cd84c64f372099eea55aae3b3410R54

Reply:
Thanks what I though - thanks :+1: